Add definition for XSD files (#33)
magento/glossary: src/terms/xsd.md

---
  title: "XSD"
  wordClasses:
    - "noun"
  tags:
    - "programming"
  synonyms:
    - "XML Schema Definition"
  relatedTerms: []
---

> Abbreviation for "XML Schema Definition" specifies how to formally describe the elements in an Extensible Markup Language ([XML](https://glossary.magento.com/xml)) document.

Reviewer: ```suggestion
Abbreviation for "XML Schema Definition". This file describes the elements in an Extensible Markup Language ([XML](https://glossary.magento.com/xml)) document.
```
